plugin/core/rpc.py

```diff
@@ -28,35 +25,8 @@ def format_request(payload: 'Dict[str, Any]') -> str:
     return json.dumps(payload, sort_keys=False)
 
 
-def attach_tcp_client(tcp_port: int, process: 'subprocess.Popen', settings: Settings) -> 'Optional[Client]':
```

Comment:
I think this is old code that was not reviewed before refactoring (probably tcp_mode=host related).
It looks like `set_transport_failure_handler` is no longer hooked up for TCP clients.
TCP servers can also log on `stdout` and `stderr`, think that also has gone missing?
Perhaps we should take that fix outside of this PR.

Reply:
I would like to keep this function removed, otherwise I would have to modify it for an extra `log_callback` parameter for `log_stream` while it's unused.